src/app/design/adminhtml/default/default/template/zendesk/config/button-signup.phtml

```diff
@@ -17,7 +17,7 @@
 ?>
 <script type="text/javascript">
     function postToZendesk() {
-        var form = new Element('form', {method: 'post', action: '<?php echo $this->getPostUrl(); ?>'});
+        var form = new Element('form', {method: 'get', action: '<?php echo $this->getPostUrl(); ?>'});
```

Comment:
Why did this change? I thought we needed a POST call to kick off the next stage of the signup process?